app/jobs/sync/launch_token_refresh_jobs_job.rb

```diff
@@ -0,0 +1,13 @@
+class Sync::LaunchTokenRefreshJobsJob
+  include Sidekiq::Worker
+
+  def perform
+    UpholdConnection.with_details.all.each do |uphold_connection|
```

Comment:
```suggestion
    UpholdConnection.with_details.find_each do |uphold_connection|
```

The `all` forces a fetch from the database whereas `find_each` properly batches for `each`.